docs/api/browser-window.md

```diff
@@ -1356,6 +1356,14 @@ removed in future Electron releases.
 **Note:** The BrowserView API is currently experimental and may change or be
 removed in future Electron releases.
 
+#### `win.setForwardMouseMessages(forward)` _Windows_
```

Comment:
Originally I wasn't sure if I could tamper with existing API, but an optional argument that defaults to false should not break anything and I agree that it's cleaner this way.

Examples are always good. Where would the example belong? My very simple test application is mostly just a matter of ignoring/acknowledging mouse input as I enter/leave a canvas respectively. Cleaned up it should make a good example if there is a good way to put it.